### ISSUE TITLE
Update request loop. 

### ISSUE THREAD

**Issue by mkrier** [BiglySoftware/BiglyBT#1853]
Ubuntu 18.04
Bigly version 2.2.0.2

It says there's a new version and I need to restart now or later.  

I say ok.

It quits and then asks me for my root password.

I give it.  

It restarts and...

Says there's a new version and I need to restart now or later.

Lather, rinse, repeat. 

I've tried this several times over the past 2 years.  It never stops asking me.  So now every time I just say "restart later".

**Comment by parg** [comment]
Did you install 2.2.0.2 from http://www.biglybt.com/ ? If so then try grabbing the latest installer from there and running it.

**Comment by mkrier** [comment]
Yes I did.  If I run the installer won't that make them run in parallel? I'll have two versions?  And/or won't I lose my download history and settings?

**Comment by parg** [comment]
The installer should only overwrite the application, not your configuration data. You can always make a backup of ~/.biglybt first, that is where all the config data is stored. 

In the unlikely event that the installer prompts you to migrate data from an existing Vuze installation make sure you say NO as this would overwrite config files.

**Comment by mkrier** [comment]
But will the new version have the same bug?  Is anyone here going to test to see if the bug is still there?

I guess I don't see the point unless the new version has some fantastic feature.  (Such as having the progress bar be a different color for seeding than stopped; my number one request, and it's easy to implement.)  Actually, is this where I post feature requests? Should I start a new post for it?  

thanks

**Comment by parg** [comment]
Features, bug fixes, I don't know about you but I generally keep my software updated. You can disable auto-updates under Options->Interface->Start if you're happy with where you are.

If you want to request a feature then the best bet would be to create a new issue here.

Thanks!

**Comment by parg** [comment]
I could make stopped/queued torrents show grey in the progress bar, how about that?

**Comment by mkrier** [comment]
Sure that'd be great.  (There really isn't already an if trap when you set it to "SEEDING" that you could change the bar color?)  Anyway close enough, that'd be great thanks.

**Comment by parg** [comment]
Beta B22 is available with that change - you would need to join the beta program to get it (Help menu option)

**Comment by mkrier** [comment]
Oh I see it. Yea I'll just wait for the release.  Btw I'd contribute if I still wrote java anymore. (Developer for 22 years here).

**Comment by parg** [comment]
yah, coding's for fools
